app/models/checklist_source_data_criteria.rb

```diff
@@ -117,7 +117,7 @@ def compare_field_values(data_criteria, criteria)
   end
 
   # searches an array of valuesets for a code
-  def code_in_valuesets(valuesets, input_code)
-    !HealthDataStandards::SVS::ValueSet.where('concepts.code' => input_code).in(oid: valuesets).empty?
+  def code_in_valuesets(valuesets, input_code, bundle_id)
+    !HealthDataStandards::SVS::ValueSet.where('concepts.code' => input_code).in(oid: valuesets, bundle_id: bundle_id).empty?
```

Comment:
Should the `bundle_id: bundle_id` be in the `where` statement, rather than the `in` statement?